ui/src/views/tools/ImportUnmanagedInstance.vue

```diff
@@ -111,7 +111,7 @@
                   </a-select-option>
                 </a-select>
               </a-form-item>
-              <a-form-item name="templateid" ref="templateid" v-if="cluster.hypervisortype === 'KVM' && !selectedVmwareVcenter && !isDiskImport && !isExternalImport">
+              <a-form-item name="templateid" ref="templateid" v-if="cluster.hypervisortype === 'VMware' || (cluster.hypervisortype === 'KVM' && !selectedVmwareVcenter && !isDiskImport && !isExternalImport)">
```

Comment:
this can be some kind of `[].contains()` structure, right?
```suggestion
              <a-form-item name="templateid" ref="templateid" v-if="['VMware', 'KVM'].contains(cluster.hypervisortype) && !selectedVmwareVcenter && !isDiskImport && !isExternalImport)">
```
?

Reply:
Sorry this is not the same condition, we always want to display if for VMware but for KVM only if `!selectedVmwareVcenter && !isDiskImport && !isExternalImport`

Reply:
I had the same question, thanks @nvazquez

Reply:
ha yes thanks @nvazquez. sorry about my crooked eyes. Maybe you can extract the condition to a method or otherwise make it a little more reader friendly?

Reply:
@nvazquez @DaanHoogland I'm merging this as it is as default template import is blocking testing for #8426 . Please open a new PR if the refactoring is needed here :pray: